console/src/view/async_ops.rs

```diff
@@ -114,24 +114,16 @@ impl TableList for AsyncOpsTable {
                 .filter_map(move |async_op| {
                     let async_op = async_op.upgrade()?;
                     let async_op = async_op.borrow();
-
-                    let parent_id_str = match async_op.parent_id() {
-                        Some(id) => {
-                            format!("{}", id)
-                        }
-                        None => "n/a".to_string(),
-                    };
-
                     let task_id = async_op.task_id()?;
-                    let task_name = state
+                    let task = state
                         .tasks_state()
                         .task(task_id)
                         .and_then(|t| t.upgrade())
-                        .and_then(|t| t.borrow().name().map(String::from));
+                        .map(|t| t.borrow().short_desc().to_owned());
 
-                    let task_str = match task_name {
-                        Some(name) => format!("{} ({})", task_id, name),
-                        None => format!("{}", task_id),
+                    let task_str = match task {
+                        Some(task_str) => task_str,
+                        None => async_op.task_id_str().to_owned(),
                     };
```

Comment:
nit/tioli: could just be
```suggestion
                    let task_str = task.unwrap_or_else(||  async_op.task_id_str().to_owned());
```